server/apps/distributed-app/docs/modules/ROOT/pages/configure/usersrepository.adoc

```diff
@@ -36,6 +36,9 @@ recipient local part than as login for different protocols.
 |user's name. Allow a user to access to the https://tools.ietf.org/html/rfc4616#section-2[impersonation command],
 acting on the behalf of any user.
 
+| verifyFailureDelay
+| Delay after a failed authentication attempt with an invalid user name or password. Duration string defaulting to seconds. No delay if 0 or omitted.
```

Comment:
Please indicate units being used, which unit is used if none is specified, and also what is the default value (`0`).

Reply:
Also add this property within `src/site/xdoc/server/config-usersrepository.xml`...

Reply:
Uh, it does say all these things...? But maybe some examples would be more helpful. i.e.: 2, 2s, 2000ms
And of course I always forget about the other documentation location...

Reply:
Pushed, hope this is better?